new/LIT6378HymnEstifanos.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6378HymnEstifanos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እምጥበብከ፡ ስፉሕ፡ ወእምጸጋከ፡ ረቂቅ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾƎmṭǝbabǝka sǝfuḥ waʾǝmṣǝggāka raqiq</title>
```

Comment:
Please separate the prepositions with -.

Reply:
OK. I think that I have done this correctly, with the - after the ወ as well